### ISSUE TITLE
Bug: While creating a new event type duration exceed error 

### ISSUE THREAD

**Issue by Lakszy** [calcom/cal.com#12509]
### Issue Summary

When attempting to add a new event type and typing a duration that exceeds 15,000,000,000,000 minutes, an error popup with the message "BAD_REQUEST" is displayed.

### Steps to Reproduce

1. Navigate to the event types section.
2. Click on "Add New."
3. In the modal that opens, type a duration that exceeds 15000000000000 minutes.

### Actual Results

The system displays an error popup with the message "BAD_REQUEST."

### Expected Results

The system should have a validation mechanism with an upper limit for the duration input, and when the user exceeds this limit, a clear and user-friendly error message should be displayed.

### Technical details

- Browser version: Chrome is up to date
Version 119.0.6045.160 (Official Build) (64-bit)

### Evidence

This issue was tested by following the provided steps. Here is a screenshot of the error popup displaying "BAD_REQUEST":

![Error Screenshot](https://github.com/calcom/cal.com/assets/117038630/2694a557-b0c7-44d9-ad08-5a226e5879f3)

**Comment by Udit-takkar** [comment]
@Lakszy okay do you have a limit in mind ?

**Comment by Udit-takkar** [comment]
@Lakszy okay we can set the limit in zod schema.

**Comment by crediblebilal** [comment]
What maximum limit should we set for it?

**Comment by Lakszy** [comment]
Hi @Udit-takkar, can you help me with contributing via a pull request?

**Comment by Lakszy** [comment]
Hi, I'm @PeerRich. This is my first contribution and I was wondering if you could please assign me the task...... : )

**Comment by Lakszy** [comment]
I suggest setting the maximum limit to 121 hours, which is 1 hour greater than the world record for the longest meeting.

7260 minutes 🔢 :)?

**Comment by Lakszy** [comment]
> What maximum limit should we set for it?

I suggest setting the maximum limit to 121 hours, which is 1 hour greater than the world record for the longest meeting.
7260 minutes 🔢 :)?

**Comment by mohit-sankhla2002** [comment]
The bad request thing comes from here I think. The problem is that it exceeds the Int limit. 
<img width="799" alt="Screenshot 2023-12-02 at 11 56 50 PM" src="https://github.com/calcom/cal.com/assets/100336545/5d2b3bf0-3483-4043-a9a7-e7acfa3b106f">

**Comment by Udit-takkar** [comment]
> The bad request thing comes from here I think. The problem is that it exceeds the Int limit. <img alt="Screenshot 2023-12-02 at 11 56 50 PM" width="799" src="https://private-user-images.githubusercontent.com/100336545/287467721-5d2b3bf0-3483-4043-a9a7-e7acfa3b106f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE1NDIxOTYsIm5iZiI6MTcwMTU0MTg5NiwicGF0aCI6Ii8xMDAzMzY1NDUvMjg3NDY3NzIxLTVkMmIzYmYwLTM0ODMtNDA0My1hOWE3LWU3YWNmYTNiMTA2Zi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIwMlQxODMxMzZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hZGZjNzNkYmUzMTliMDI3NzczZDVkNGI4ZDQ2M2JhYzYwNGI1ODQ3NmUyZDQyNzcwZTBlMTIwYjkyY2FkNmQzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.7_rO9wbSW5kTLShN-gjppf3Vh4EVfi0U22aYHcn3Vl4">

yes https://github.com/calcom/cal.com/pull/12531#pullrequestreview-1755135046

**Comment by CarinaWolli** [comment]
This is fixed: 
<img width="541" height="143" alt="Image" src="https://github.com/user-attachments/assets/bdce7509-9353-40aa-9bad-12f8397a3dec" />
